alaoweb/2021.alaoweb.org
# auto-scroll schedule page to today's date (if today is on the calendar)

closes #16 

Mechanism:
* adds `div id="yyyy_mm_dd"` to top of each day schedule
* javascript determines location of the id, scrolls down to it, leaving room for the header

To test: 
1. In this PR, check out the /schedule page -- it should load normally  because today's date is not on the calendar
2. Simulate what happens when the date is on the calendar:
  * in `js/scripts.js`, uncomment line 17: `// today = '2021_10_29';` to pretend that today is Oct 29
  * on command line, recompile javascript with this modification to today's date: 
```
cd ./automation/minifying
./minify_js.sh
```
  * confirm that /js/scripts.min.js has updated (check time stamp)
  * reload schedule page -- it should scroll down to Oct 29
